recipes/crispector/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.0.2b6" %}
+
+package:
+  name: crispector
+  version: "{{ version }}"
+
+build:
+  number: 0
+  entry_points:
+    - crispector = crispector.cli:main
+  noarch: generic
+  script: python -m pip install . --no-deps -vv
+
+
+source:
+  url: https://github.com/YakhiniGroup/crispector/archive/v{{ version }}.tar.gz
+  sha256: ddbfea5ee605e618266ba0310fa52323d2af66c9626a1325754ff3a1e6af7475
+
+requirements:
+  host:
+    - python 3.7
+    - pip
+    - setuptools
+  run:
+    - python 3.7
+    - biopython >=1.74
+    - fastp
+    - python-edlib
+    - click >=7.0
+    - pyyaml >=5.1.2
+    - matplotlib-base >=3.1.2
+    - scipy >=1.2.1
+    - numpy >=1.12.1
+    - seaborn >=0.9.0
+    - plotly >=4.3.0
+    - pandas >=0.24.2
+    - statsmodels
+    - jinja2
+
+extra:
```

Comment:
The `extra:` section should be the last in the recipe (I don’t think this is documented anywhere, but it’s nicer to be consistent with the other recipes).

Reply:
Done